lua/wire/stools/vthruster.lua

```diff
@@ -55,17 +55,16 @@ function TOOL:LeftClick( trace )
 			self:ReleaseGhostEntity()
 
 			self:SetObject(1, trace.Entity, trace.HitPos, trace.Entity:GetPhysicsObjectNum(trace.PhysicsBone), trace.PhysicsBone, trace.HitNormal)
-			self:SetObject(2, ent, trace.HitPos, Phys, 0, trace.HitNormal)
+			self:SetObject(2, ent, trace.HitPos, ent:GetPhysicsObject(), 0, trace.HitNormal)
 			self:SetStage(1)
 		end
 	else
 		if (CLIENT) then return true end
 
 		local anchor, wire_thruster = self:GetEnt(1), self:GetEnt(2)
 		local anchorbone = self:GetBone(1)
-		local normal = self:GetNormal(1)
 
-		local const = WireLib.Weld(wire_thruster, anchor, trace.PhysicsBone, true, false)
+		local const = WireLib.Weld(wire_thruster, anchor, anchorbone, true, false)
```

Comment:
This was only supposed to be cleanup, but it turns out this fixes a bug where attaching a vector thruster to a model with multiple bones would always weld to bone 0.